hugegraph-test/src/main/java/com/baidu/hugegraph/unit/core/RangeTest.java

```diff
@@ -26,7 +26,7 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import com.baidu.hugegraph.backend.store.BackendTable.ShardSpliter.Range;
+import com.baidu.hugegraph.backend.store.BackendTable.ShardSplitter.Range;
 import com.baidu.hugegraph.backend.store.Shard;
 import com.baidu.hugegraph.testutil.Assert;
 
```

Comment:
You can't see the diff for the next item because in `master`, it has a null byte.
This commit should resolve that:
https://github.com/hugegraph/hugegraph/commit/f16c8fa749dc5de6214e5c73a75a68001ae72af3
And here's the commit containing my primary change:
https://github.com/hugegraph/hugegraph/commit/324617ea6bf0917664d671e4f4cb4f81e53d953b

Reply:
looks good